test_util/provide_aws.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
```

Comment:
Why not `test_util/aws.py`?

Why do we need the `#!/usr/bin/env python3` here. This doesn't have a main().